src/components/kroniskkrav/KroniskKrav.test.tsx

```diff
@@ -1,16 +1,78 @@
 import React from 'react';
-import { render, cleanup } from '@testing-library/react';
+import { render, cleanup, screen } from '@testing-library/react';
 import { axe } from 'jest-axe';
+import userEvent from '@testing-library/user-event';
 
 import KroniskKrav from './KroniskKrav';
 import { MemoryRouter } from 'react-router-dom';
-import { ArbeidsgiverProvider } from '@navikt/helse-arbeidsgiver-felles-frontend';
+import { ArbeidsgiverProvider } from '../../context/arbeidsgiver/ArbeidsgiverContext';
+import { Organisasjon } from '@navikt/bedriftsmeny/lib/organisasjon';
+import testFnr from '../../mockData/testFnr';
+
+const arbeidsgivere: Organisasjon[] = [
```

Comment:
Kan vurdere å flytte testdataene til mockData mappen for økt lesbarhet.

Reply:
Trenger ikke det. Er jo der allerede :-)